modules/core/include/opencv2/core/hal/intrin.hpp

```diff
@@ -1226,6 +1226,105 @@ namespace CV__SIMD_NAMESPACE {
 //! @endcond
 
 
+//! @name Exponential
+//! @{
+//! @brief Calculates exponent of every vector element
+    inline v_float32 v_exp(const v_float32 &x) {
```

Comment:
@WanliZhong, we _must_ pay credits to the original author. Please, add the original license for this implementation, as it was done here: https://github.com/vpisarev/ficus/blob/3c9a8b78f49e17489c5e1fd6dd5dd487348c99c2/lib/NN/ficus_nn_common.h#L216. Together with the license, please, put the link to https://github.com/aff3ct/MIPP/blob/master/src/math/neon_mathfun.h